persistence/src/main/java/io/nobt/persistence/cashflow/expense/ExpenseEntity.java

```diff
@@ -37,6 +37,9 @@
     @Column(name = "date", nullable = false)
     private LocalDate date;
 
+    @Column(name = "deleted", nullable = false)
```

Comment:
Wenns hier nullable=false ist, sollte das auch im Migrationskript so in die DB geschrieben werden. Ich würde vielleicht auch `insertable=false` setzen. Ein hinzufügen einer gelöschten Rechnung macht keinen Sinn.